### PR TITLE
feat: add todo list guide

### PR DESCRIPTION
**Pull request by HomelessDinosaur**

### CODE REVIEW

File: docs/guides/serverless-api-with-nextjs.md [hunk 1]
```diff
@@ -0,0 +1,515 @@
+# Nitric To-Dos
+
+In this tutorial, you'll create a [Next.js](https://nextjs.org/) application that connects to a task list [Nitric API](https://nitric.io/docs/apis) and a task list [Nitric Collection](https://nitric.io/docs/collections). We will also be using [Tailwind CSS](https://tailwindcss.com/) for styling.
```
Comment (asalkeld) [inline docs/guides/serverless-api-with-nextjs.md:3]: this sounds odd, I much prefer the wording in the overview

Comment (HomelessDinosaur) [reply docs/guides/serverless-api-with-nextjs.md]: Removed it, it was duplicating the overview section

File: doc.config.json [hunk 2]
```diff
@@ -94,6 +94,16 @@
                 "path": "/docs/contributions.md"
               }
             ]
+          },
+          {
+            "title": "Guides",
+            "disableAutoPrefix": true,
+            "routes": [
+              {
+                "title": "API with Nextjs",
```
Comment (davemooreuws) [inline doc.config.json:103]: Change to Next.js and search the doc in case there are others

**Review by davemooreuws — CHANGES_REQUESTED** [review]

File: doc.config.json (hunk 2 above)
Comment (raksiv) [reply doc.config.json:103]: Perhaps the title should be something like "Build a todo list with Nextjs" ?

File: docs/guides/serverless-api-with-nextjs.md [hunk 3]
```diff
@@ -0,0 +1,512 @@
+## Overview
+
+This guide shows you how to create a simple To-do list in Nextjs with an API backed by serverless functions. It also uses collections for data persistence.
```
Comment (raksiv) [inline docs/guides/serverless-api-with-nextjs.md:3]: Can we change to call out that we are building with Nextjs and Nitric.

E.g. change serverless to Nitric functions

File: docs/guides/api-with-nextjs.md [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+  return ctx;
+});
+```
+
+### Set up API proxy
+
+Start by create your `.env` file by renaming the `.env.example` file:
+
+```
+mv web/apps/.env.example web/apps/.env
```
Comment (raksiv) [inline docs/guides/api-with-nextjs.md:435]: The .env file is checked in to the repo instead of the .env.example

Comment (davemooreuws) [reply docs/guides/api-with-nextjs.md:435]: I will take over this one and fix it up

File: docs/guides/api-with-nextjs.md [hunk 5]
```diff
@@ -0,0 +1,512 @@
...
+- The cloud of your choice:
+  - [AWS](https://aws.amazon.com)
+  - [GCP](https://cloud.google.com)
+  - [Azure](https://azure.microsoft.com)
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/)
+- [Nitric CLI](https://nitric.io/docs/installation)
+
```
Comment (raksiv) [inline docs/guides/api-with-nextjs.md:19]: Include Next in the prereq's since Nitric CLI is included for the api side.

**Comment by raksiv** [comment]
Looking at the UI - Could see a way to remove list items, or mark them complete.
We probably want this functionality.

**Comment by davemooreuws** [comment]
> Looking at the UI - Could see a way to remove list items, or mark them complete. We probably want this functionality.

I have added this for tasks now, with a popover menu

**Review by raksiv — APPROVED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**